src/OpenGraph.php

```diff
@@ -14,7 +14,12 @@ public function fetch($url, $allMeta = null, $lang = null)
          * parsing starts here:.
          */
         $doc = new DOMDocument();
-        $doc->loadHTML('<?xml encoding="utf-8" ?>'.$html);
+
+        try {
+            $doc->loadHTML('<?xml encoding="utf-8" ?>'.$html);
+        } catch (\Exception $exception) {
+            //catch possible errors due to empty or malformed HTML
```

Comment:
@alinmiron Can we throw this exception or at least log it? so that the consumer gets to know about it.

Reply:
@shweshi done. pushed in master. it should be visible aready. thanks

Reply:
@alinmiron Thanks